purchases/build.gradle.kts

```diff
@@ -0,0 +1,197 @@
+    androidTestImplementation(libs.mockk.android)
+    androidTestImplementation(libs.mockk.agent)
+}
+
+tasks.dokkaHtmlPartial.configure {
+    dokkaSourceSets {
+        named("customEntitlementComputation") {
+            suppress.set(true)
+        }
+        named("defaults") {
+            dependsOn("main")
+            reportUndocumented.set(true)
+            includeNonPublic.set(false)
+            skipDeprecated.set(true)
+            externalDocumentationLink {
+                url.set(
+                    uri("https://developer.android.com/reference/package-list").toURL(),
+                )
+            }
+            sourceLink {
+                localDirectory.set(
+                    file("src/main/kotlin"),
+                )
+                remoteUrl.set(
+                    uri("https://github.com/revenuecat/purchases-android/blob/main/purchases/src/main/kotlin").toURL(),
+                )
+                remoteLineSuffix.set("#L")
+            }
+            sourceLink {
+                localDirectory.set(
+                    file("src/main/java"),
+                )
+                remoteUrl.set(
+                    uri("https://github.com/revenuecat/purchases-android/blob/main/public/src/main/java").toURL(),
+                )
+                remoteLineSuffix.set("#L")
+            }
+        }
+        named("main") {
+            reportUndocumented.set(true)
+            includeNonPublic.set(false)
+            skipDeprecated.set(true)
+
+            // This package exclusively contains symbols annotated with @InternalRevenueCatAPI, for which no
+            // documentation is generated due to our dokka-hide-internal plugin. However, by default Dokka still
+            // generates an empty page for the package. This avoids that.
+            perPackageOption {
+                matchingRegex.set("com\\.revenuecat\\.purchases\\.paywalls\\.components.*")
+                suppress.set(true)
+            }
+            externalDocumentationLink {
+                url.set(
+                    uri("https://developer.android.com/reference/package-list").toURL(),
+                )
+            }
+            sourceLink {
+                localDirectory.set(
+                    file("src/main/kotlin"),
+                )
+                remoteUrl.set(
+                    uri(
+                        "https://github.com/revenuecat/purchases-android/blob/main/purchases/src/main/kotlin",
+                    ).toURL(),
+                )
+                remoteLineSuffix.set("#L")
+            }
+            sourceLink {
+                localDirectory.set(file("src/main/java"))
+                remoteUrl.set(
+                    uri("https://github.com/revenuecat/purchases-android/blob/main/public/src/main/java").toURL(),
+                )
+                remoteLineSuffix.set("#L")
+            }
+        }
+    }
+}
+
+// Remove afterEvaluate
+// after https://github.com/Kotlin/kotlinx-kover/issues/362 is fixed
```

Comment:
Looks like this was pretty outdated 😅 We should try to fix this at some point as well.